include/tfm/platform_builtin_key_loader_ids.h

```diff
@@ -11,7 +11,7 @@
 extern "C" {
 #endif
 
-#define TFM_BUILTIN_MAX_KEY_LEN 96
+#define TFM_BUILTIN_MAX_KEY_LEN 32
```

Comment:
Appears we may have been wasting bytes before.

Reply:
discussed offline